tasks/deployment/stark-proxy-v3.ts

```diff
@@ -0,0 +1,23 @@
+import { types } from 'hardhat/config';
+
+import mainnetAddresses from '../../src/deployed-addresses/mainnet.json';
+import { hardhatTask } from '../../src/hre';
+import { log } from '../../src/lib/logging';
+import { deployStarkProxyV3 } from '../../src/migrations/deploy-stark-proxy-v3';
+
+hardhatTask('deploy:stark-proxy-v3', 'Deploy the StarkProxyV2 contracts.')
+  .addParam('liquidityStakingAddress', 'Previously deployed liquidity staking address', mainnetAddresses.liquidityStaking, types.string)
+  .addParam('merkleDistributorAddress', 'Previously deployed merkle distributor address', mainnetAddresses.merkleDistributor, types.string)
+  .addParam('starkPerpetualAddress', 'Address of DYDX stark perpetual', mainnetAddresses.starkPerpetual, types.string)
+  .addParam('dydxCollateralTokenAddress', 'Address of collateral token for DYDX stark perpetual', mainnetAddresses.dydxCollateralToken, types.string)
+  .addOptionalParam('ownerRoleAddress', 'Address to give owner role to', undefined, types.string)
```

Comment:
nit: I think this parameter should be required, since we shouldn't deploy the stark proxy contracts without knowing what address to assign `OWNER_ROLE` to.